.golangci.yml

```diff
@@ -42,6 +43,11 @@ linters-settings:
       - G306
       - G402
       - G404
+  depguard:
+    list-type: denylist
+    include-go-root: true
+    packages-with-error-message:
+      - io/ioutil: "deprecated in 1.16 (see https://go.dev/doc/go1.16#ioutil)"
```

Comment:
IIUC, this entire PR is only adding manual deprecation of `io/ioutil`, and not any additional packages... 

If that's true, then IMO this isn't really worth worrying about, as the `ioutil` pkg [was officially marked as deprecated in `1.19`](https://github.com/golang/go/pull/51961). One of the standard golangci linters flags deprecated functions, so once `containerd` bumps to 1.19 `ioutil` will start getting auto-flagged w/o needing any additional configuration.

So adding it now may protect for a couple of months, but after that merely adds confusion until it gets ripped out... 

Anyway, not something I feel strongly about, but I saw this and thought I'd mention it as something to consider...

Reply:
Ah, yep, haven't seen that PR with deprecations.
So its only relevant until we upgrade to `1.19`.
I don't feel strongly about this either, let's see what other ppl think.

Reply:
Seems like we have the PR to move to 1.19 now; should we close this?

Reply:
Yep, we can now close this one.